framework/src/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -925,7 +927,7 @@ private Range calculateRangeUnaryOp(
                 break;
             case GREATER_THAN_EQ:
                 thenRightRange = rightRange.refineLessThanEq(leftRange);
-                thenLeftRange = leftRange.refineGreaterThan(rightRange);
+                thenLeftRange = leftRange.refineGreaterThanEq(rightRange);
```

Comment:
Please do.  Since this PR is likely to take a few days before it is merged, it would be good to have this fix change in master sooner.